exist-distribution/pom.xml

```diff
@@ -271,12 +271,12 @@
             <version>${project.version}</version>
             <scope>runtime</scope>
         </dependency>
-        <dependency>
+        <!--dependency>
```

Comment:
This is fine, but should be done in a major release, e.g. 6.0.0 Why not also remove the module itself?

Reply:
good point. I just wanted to get rid of it asap :-)

Reply:
Okay, well it's up to you then ;-)

Reply:
I am for removing it and rather document how to get it if needed.
I would consider this a security related issue and therefore would allow this feature removal as part of a feature release.